src/Form/DropdownField.jsx

```diff
@@ -15,12 +15,19 @@ export const DropdownField = ({ options, ...props }) => {
     [field, options]
   )
 
-  const selectedLabel = selectedOption ? selectedOption.label : undefined
+  const selectedLabel =
+    selectedOption && selectedOption.disabled !== true
```

Comment:
Isn't `!selectedOption.disabled` enough? If someone passes `disabled={1}` it's still disabled right?

Reply:
The disabled option should be very specific. If someone passes `disabled='no'`, that option should not be disabled.

Reply:
you are both right, maybe go with the middle solution where if set (option !== undefined) then check for truthiness (disabled == true)